packages/flutter/lib/src/widgets/widget_state.dart

```diff
@@ -263,12 +256,12 @@ abstract class WidgetStateColor extends Color implements WidgetStateProperty<Col
   /// Creates a [WidgetStateColor] from a [WidgetStateMap<Color>].
   ///
   /// {@macro flutter.widgets.WidgetStateProperty.fromMap}
-  ///
-  /// If used as a regular color, the first key that matches an empty
-  /// [Set] of [WidgetState]s will be selected.
+  /// It should only be used with widgets that document
+  /// support for [WidgetStateColor], like [TimePickerThemeData.dayPeriodColor]
+  /// (if used as a [Color], it will be interpreted as a transparent black).
```

Comment:
Is this true? Or will it resolve against no states, or `any` if set in the mapper?

Reply:
The `_WidgetStateColorMapper` constructor will call `super(0)` regardless of the map's contents 👍